server/services/userService/index.test.ts

```diff
@@ -14,13 +14,15 @@ describe('User service', () => {
       hmppsAuthClient = new HmppsAuthClient(null) as jest.Mocked<HmppsAuthClient>
       userService = new UserService(hmppsAuthClient)
     })
+
```

Comment:
No actual changes? Is it a linter doing this?

Reply:
I think it's just a line-break between the `it()` blocks - not sure whether it's something I've done manually for readability or whether it's the linter.